### DISCUSSION TITLE
jmx-metrics for weblogic ?

### DISCUSSION THREAD

**Discussion by rhavenInc** [open-telemetry/community#2101] [Community Discussions]
new to open telemetry and doing some research to see if it is possible to collect some jmx metrics from Weblogic.

the only supported platform atm is https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/jmx-metrics/README.md#target-systems

Are there plans to include WL?

**Comment by trask** [comment]
hi @rhavenInc! can you open an issue in https://github.com/open-telemetry/opentelemetry-java-contrib where we can tag folks who work on that component?
